.github/workflows/master.yml

```diff
@@ -27,21 +26,21 @@ jobs:
         # Kinesis tests depends on external Amazon kinesis service.
         # Note that the modules below are from sparktestsupport/modules.py.
         modules:
-          - |-
+          - >-
```

Comment:
By doing this, it treats newlines as spaces, which  It makes the title prettier. Without this, [actions/upload-artifact](https://github.com/actions/upload-artifact) fails for unknown reason.